### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by anondev-sudo** [Sorok-Dva/freenom-auto-renew-domains#7]
```
> freenom-autorenew-domains@1.0.1 start
> node .

Connected to the freenoms-domains database.
[INIT] Failed TimeoutError: Timed out after 30000 ms while trying to connect to the browser! Only Chrome at revision r901912 is guaranteed to work.
    at Timeout.onTimeout (/home/container/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:208:20)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7)


```
Got this error

**Comment by anondev-sudo** [comment]
Fixed by updating puppeteer to the latest version, now getting

```
Connected to the freenoms-domains database.
Domain Checker - Freenom
Username complete
Password complete
connected
[renew] Error ReferenceError: messages is not defined
    at /home/container/config/freenom.js:122:9
    at async Promise.all (index 0)
    at async Object.renewFreeDomains (/home/container/config/freenom.js:92:7)
    at async Object.init (/home/container/config/freenom.js:48:7)
    at async run (/home/container/app.js:14:3)
Scrap finished for https://my.freenom.com/domains.php?a=renewals
```

**Comment by anondev-sudo** [comment]
Fixed it by defining messages (also going to make PR to fix), but now getting errors like this
```

message length > 2000 - **domain.tk** _(free)_: **185** days left.
  Auto renewal is **enabled**. 


{
  "global": false, 
  "message": "You are being rate limited.", 
  "retry_after": 74
}
```
Is the ratelimited message normal even if it doesnt tried to renew any domain, there is none that will expire in less than 14 days.
@Sorok-Dva

**Comment by Sorok-Dva** [comment]
> Is the ratelimited message normal even if it doesnt tried to renew any domain, there is none that will expire in less than 14 days.

Yes it is normal, each domain you have = one discord message, so if you have tons of domains, it being rate limited by discord, but normally this problem was fixed with this PR #4 

Can you confirm you have this error with this PR ?

**Comment by anondev-sudo** [comment]
I will try soon
